pymc3/tests/test_distributions.py

```diff
@@ -1328,18 +1328,6 @@ def test_dirichlet(self, n):
             Dirichlet, Simplex(n), {"a": Vector(Rplus, n)}, dirichlet_logpdf
         )
 
-    @pytest.mark.parametrize("n", [3, 4])
-    def test_dirichlet_init_fail(self, n):
-        with Model():
-            with pytest.raises(
-                ValueError, match=r"All concentration parameters \(a\) must be > 0."
-            ):
-                _ = Dirichlet("x", a=np.zeros(n), shape=n)
-            with pytest.raises(
-                ValueError, match=r"All concentration parameters \(a\) must be > 0."
-            ):
-                _ = Dirichlet("x", a=np.array([-1.0] * n), shape=n)
```

Comment:
As I wrote above, I think users shouldn't be allowed to enter such parameters without PyMC3 erroring out explicitly

Reply:
Per [my earlier comments](https://github.com/pymc-devs/pymc3/pull/4000#discussion_r450280961), that's not possible to do in a consistent way&mdash;aside from one extreme case&mdash;without effectively maintaining two PyMC3 codebases: one that's Theano compatible, and another that circumvents (and duplicates the functionality of) Theano and works only with NumPy arrays.